### DISCUSSION TITLE
Problema invio mail

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#662] [Richieste di aiuto]
Buonasera,usando Altervista ,ho provato a configurare l'invio delle mail tramite la funzione "Usa server locale tramite PHP",ma non appena provo ad inviare l'email di prova ,mi ritrovo con l'errore 500,allego il file 

[2024-03-05_22-46.log](https://github.com/iisgiua/giuaschool/files/14501852/2024-03-05_22-46.log)

**Comment by trinko** [comment]
Il log non riporta nessun errore. 
Forse:
- non hai attivato il DEBUG
- oppure non hai rieseguito la pagina con errore dopo aver attivato il DEBUG
- o ancora, hai indicato un orario errato per l'estrazione del log, cioè successivo al momento in cui e' stata eseguita la pagina con errore.

**Reply by sempliciniluigi89** [reply]
[2024-03-06_20-06.log](https://github.com/iisgiua/giuaschool/files/14514407/2024-03-06_20-06.log)

**Comment by trinko** [comment]
Anche questo file di log non riporta la pagina di errore.
Si vedono solo le informazioni sulla pagina **Sistema > Mantenzione**
